contrib/gitian-build.py

```diff
@@ -208,7 +208,7 @@ def main():
     args.macos = 'm' in args.os
 
     # Disable for MacOS if no SDK found
-    if args.macos and not os.path.isfile('gitian-builder/inputs/MacOSX10.11.sdk.tar.gz'):
+    if args.macos and not os.path.isfile('gitian-builder/inputs/MacOSX10.14.sdk.tar.gz'):
```

Comment:
Should the gitian-build in master support compiling older releases?

Reply:
Not sure that's a burden we wanna take on. It's probably going to be more sane for those with that use-case to check out the older tag.

Reply:
Seems fine